main/templates/navbar.html

```diff
@@ -4,14 +4,22 @@
         <div class="pull-left">
           <a class="brand-logo" href="http://www.mit.edu"></a>
           <div class="divider-large">&nbsp;</div>
-          <a class="bootcamp-link" href="/">Bootcamps</a>
+          <a class="bootcamp-link" href={%  url "bootcamp-index" %}>Bootcamps</a>
           </div>
         <div class="pull-right">
           {% if authenticated %}
-          <a class="header-dashboard-link" href="/logout">
+          <a class="header-dashboard-link" href={%  url "logout" %}>
             Sign Out
           </a>
           {% endif %}
+          {% if social_auth_enabled and not authenticated %}
+          <a class="header-dashboard-link reverse" href={% url "login" %}>
```

Comment:
Can we add quotes around the `href` attributes in this file?

Reply:
oops, fixed